__tests__/components/HTMLBlock.test.jsx

```diff
@@ -8,7 +8,7 @@ const HTMLBlock = require('../../components/HTMLBlock')(sanitize);
 describe('HTML Block', () => {
   beforeEach(() => {
     global.window = true;
-    global.mockFn = jest.fn(() => console.log('RAN CUSTOM SCRIPT'));
+    global.mockFn = jest.fn();
```

Comment:
Removed this console because it's not being used and was throwing an ESLint warning.